### PR TITLE
[NEW][10.0] pos_cashbox module

### PR DESCRIPTION
**Pull request by ilmir-k**
[NEW][10.0] pos_cashbox module

### CODE REVIEW

**Comment by ilmir-k** [comment]
issue for unitest #662

**Review by Railia — APPROVED** [review]

**Review by Railia — APPROVED** [review]
👍

File: pos_cashbox/views/pos_cashbox_view.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                    <button name="open_backend_cashbox" type="object" string="Open CashBox"
+                            attrs="{'invisible':['|','|', ('iface_cashdrawer', '=', False), ('cash_control', '=', False), ('state', '!=', 'opening_control')]}"
+                            class="oe_highlight o_wow" special="open_backend_cashbox">
+                        <i class='fa fa-archive' />
+                    </button>
+                    <button name="open_backend_cashbox" type="object" string="Open CashBox"
+                            attrs="{'invisible':['|','|', ('iface_cashdrawer', '=', False), ('cash_control', '=', False), ('state', '!=', 'closing_control')]}"
+                            class="oe_highlight o_wow" special="open_backend_cashbox">
+                        <i class='fa fa-archive' />
+                    </button>
```
Comment (yelizariev) [inline pos_cashbox/views/pos_cashbox_view.xml]: Why there are two the same buttons?

File: pos_cashbox/static/src/js/tour.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        position: "bottom"
+    }, {
+        trigger: '.o_main_content:has(.loader:hidden)',
+        content: 'waiting for loading to finish',
+        run: function () {
+            // specify empty run method to don't click on trigger
+        },
+    }
+    ]);
+});
```
Comment (yelizariev) [inline pos_cashbox/static/src/js/tour.js]: It's stupid test which opens POS, while the feature (button) is in backend

**Review by yelizariev — CHANGES_REQUESTED** [review]
Also, add copyright lines

**Comment by yelizariev** [comment]
default

File: pos_cashbox/views/pos_cashbox_template.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Dinar Gabbasov <https://it-projects.info/team/GabbasovDinar>
+License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+    <data>
+        <template id="assets_backend" name="cash_box assets" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
+                <script type="text/javascript" src="/pos_cashbox/static/src/js/cashbox.js"></script>
+                <script type="text/javascript" src="/pos_cashbox/static/src/js/tour.js"></script>
```
Comment (ilmir-k) [inline pos_cashbox/views/pos_cashbox_template.xml]: @GabbasovDinar I suppose it can be removed due to tour.js was removed

**Review by yelizariev — APPROVED** [review]
Previous requests are done

**Comment by ilmir-k** [comment]
@GabbasovDinar 2 remaining lints to be fixed

**Comment by GabbasovDinar** [comment]
@ilmir-k DONE (I sent the PR to your branch)

**Comment by ilmir-k** [comment]
@yelizariev I think it can be merged. It's ready for publication

**Comment by ilmir-k** [comment]
dva-zhduna

**Comment by yelizariev** [comment]
We need to fix pos_invoice_pay and check are there other errors or not

**Comment by GabbasovDinar** [comment]
@yelizariev We have one lint error about test only. I think we can merge it

**Pull request merged**